components/scream/src/share/tests/field_tests.cpp

```diff
@@ -530,33 +530,33 @@ TEST_CASE("multiple_bundles") {
   FieldIdentifier e_id("e", {tags, dims}, nondim, grid_name);
   FieldIdentifier f_id("f", {tags, dims}, nondim, grid_name);
 
-  FieldRequest a_req(a_id,"group1");
+  FieldRequest a_req(a_id,SL{"group1","group3"});
   FieldRequest b_req(b_id,SL{"group1"});
   FieldRequest c_req(c_id,SL{"group1","group2"});
   FieldRequest d_req(d_id,SL{"group1","group3"});
   FieldRequest e_req(e_id,SL{"group1","group2"});
-  FieldRequest f_req(f_id,SL{"group1","group3"});
+  FieldRequest f_req(f_id,SL{"group1"});
 
   GroupRequest g1_req ("group1",grid_name,Bundling::Required);
   GroupRequest g2_req ("group2",grid_name,Bundling::Required);
-  // group3 = group3 + group2
-  GroupRequest g3_req ("group3",grid_name,4,Bundling::Required,&g2_req,Relationship::Child);
+  // group3 += group2
```

Comment:
Do we mean to keep this commented line?

Reply:
Ah yeah. This line is simply explaining how group3 is "extended" by the following group request. Maybe I can improve the comment.